resources/developer-guides/daemon-rpc.md

```diff
@@ -1887,7 +1887,6 @@ Inputs:
 Outputs:
 
 * *outs* - array of structure *outkey* as follows:
-  * *amount* - unsigned int;
```

Comment:
Is this here for backward compatibility? (from when amounts were public)

Reply:
iirc we have a few instances of supporting backward compatibility for devs experimenting at different points of the blockchain

Reply:
I mean: it shouldn't be removed if its still needed for older parts of the bc

Reply:
a quick search returned amount being added in a related (but not exact) field https://github.com/monero-project/monero/commit/11dc091464a6cef41434a0bb9f8604f6151c8dc5#diff-da6ab0aecdf9547f57bb822783cae11fcfbaf7a2eae0d9cb5dbb69c8fc3b660aR282